descriptions/SDK.PathEngine.md

```diff
@@ -0,0 +1,2 @@
+[**PathEngine**](https://www.pathengine.com/overview) is a sophisticated middleware tool-kit for the implementation of intelligent agent movement, built around an advanced implementation of points-of-visibility pathfinding on 3D ground surfaces.
```

Comment:
i'm not really a fan of that suggestion but it's at least better than that quote from their website
```suggestion
[**PathEngine**](https://www.pathengine.com/overview) is a path finding SDK that was first released in 2002.
```